src/Helper/DebitHelper.php

```diff
@@ -118,6 +141,9 @@ public function createPlentyPayment($orderId, $contactBank)
         $payment->status            = Payment::STATUS_APPROVED;
         $payment->unaccountable     = 1;
         $payment->regenerateHash    = true;
+        $payment->amount            = $order->amount->invoiceTotal;
```

Comment:
Sind Mehrzweck-Gutscheineinlösungen mit dem Plugin möglich? Wenn ja, dann kann ein Teil der Rechnungssumme bereits durch den Gutschein abgeglichen sein:
```suggestion
        $payment->amount            = $order->amount->invoiceTotal - $order->amount->giftCardAmount;
```